scripts/oracle/price_feeder.py

```diff
@@ -14,6 +15,11 @@
     " tx oracle aggregate-vote abc 100uusdc,50uatom {val_addr} --from={key} "
     "--chain-id={chain_id} --fees=100000usei --gas=100000 -y --broadcast-mode=sync"
 )
+COMBINED_TMPL = (
+    " tx oracle aggregate-combined-vote abc {prevote_prices} abc {vote_prices} {val_addr} --from {key} "
+    "--chain-id={chain_id} --fees=100000usei --gas=100000 -y --broadcast-mode=sync"
+)
+
 
 KEY = "tony"
```

Comment:
shnould these variables be taken as cli arguments?

Reply:
Also, may want to make port a cli argument (with default 26657). Pretty common to override that

Reply:
updated